### DISCUSSION TITLE
avoiding bias...

### DISCUSSION THREAD

**Discussion by jmerc77** [PromtEngineer/localGPT#178] [Ideas]
I have a suggestion for the training data etc: Please do not make it politically biased (or biased in any topic ideally!) maybe opinionated prompts should show all angles of the topic in the response and state first-line that it is a matter of opinion. let them decide what to believe.

**Comment by Mathwhizdude** [comment]
You simply start out with something similar to the following in your prompt, if you want your prompt to be capture exacting numerical data, in your prompt. You must ask it to return it to format unbiased.

Example1 : Based on the Holy Bible, how does John 16 read in the King James Version versus the NIV or new international version.

Example2 : Hoka shoe.
Based on the manufacturers of the Hoka brand shoes, which manufacturer contributes the least in tonnage at raw material level. Here your source would be multiples, because you have multiple suppliers involved in the production of the Hoka brand shoe itself.

**Reply by Mathwhizdude** [reply]
Nice!!
